langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -254,6 +351,11 @@ public isolated function createComment(string content) returns Comment = @java:M
 #
 # The constructed sequence will be empty when the length of parameter `data` is zero.
 #
+# ```ballerina
+# string textData = "Hello world!";
+# xml:Text xmlText = xml:createText(textData);
```

Comment:
Shall we remove `# string textData = "Hello world!";` and instead do
```suggestion
# xml:createText("Hello world!") ⇒
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/69128d3dd418a7a5070e58b83db61f160d768e37